shpc/main/registry/remote.py

```diff
@@ -216,6 +217,20 @@ def _update_clone_cache(self):
             }
         shutil.rmtree(tmpdir)
 
+    def get_module_config_url(self, module_name):
+        """
+        Get the raw address of the config (container.yaml)
+        """
+        parts = self.url.split("/")
+        domain = parts[2]
+        if domain in self.raw_container_url_schemes:
+            t = ("/".join(parts[3:]), self.tag, module_name)
```

Comment:
@vsoch . If I do `parts = self.url.split("/", 3)` above, then this becomes `t = (parts[3], self.tag, module_name)`. Do you have a preference ?

Reply:
That seems elegant! The current way is good, but I like that one too :)